### PR TITLE
Create train_model.py

### PR DESCRIPTION
**Pull request by dquerales**
This is a script for retraining the model

### CODE REVIEW

**Comment by vargas88hugo** [comment]
> This is a script for retraining the model

Improve description of the PR

**Comment by vargas88hugo** [comment]
We need to add checks to this repository, let's work on that!

File: train_model.py [hunk 1]
```diff
@@ -0,0 +1 @@
+#this is for training a model
```
Comment (vargas88hugo) [inline train_model.py]: we should add a space between the hashtag and description, and where is the implementation?
```suggestion
# This is for training a model
from sklearn.model_selection import train_test_split
from sklearn.tree import DecisionTreeClassifier
from sklearn.metrics import accuracy_score

X, y = load_dataset()

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

model = DecisionTreeClassifier()

model.fit(X_train, y_train)

y_pred = model.predict(X_test)

accuracy = accuracy_score(y_test, y_pred)
print(f"Model Accuracy: {accuracy}")
```

Comment (dquerales) [reply train_model.py]: Perfect

**Comment by dquerales** [comment]
Your changes are now commited

**Review by vargas88hugo — APPROVED** [review]
LGTM! Very great job!

**Pull request merged**